app/src/main/java/fr/neamar/kiss/loader/LoadAppPojos.java

```diff
@@ -88,13 +89,15 @@ protected ArrayList<AppPojo> doInBackground(Void... params) {
         return apps;
     }
 
-    private AppPojo createPojo(UserHandle userHandle, String packageName, String activityName, CharSequence label, Set<String> excludedAppList, Set<String> excludedFromHistoryAppList) {
+    private AppPojo createPojo(UserHandle userHandle, String packageName, String activityName, CharSequence label, Set<String> excludedAppList, Set<String> excludedFromHistoryAppList, Set<String> excludedShortcutsAppList) {
         String id = userHandle.addUserSuffixToString(pojoScheme + packageName + "/" + activityName, '/');
 
         boolean isExcluded = excludedAppList.contains(AppPojo.getComponentName(packageName, activityName, userHandle));
         boolean isExcludedFromHistory = excludedFromHistoryAppList.contains(id);
+        // TODO: are we checking for 'packageName' or 'id'? Test
```

Comment:
it's 'packageName' 😉

Reply:
Whoops sorry, meant to take this out :eyes: